experimental/framework/fluid-static/README.md

```diff
@@ -45,20 +45,20 @@ const container = await Fluid.getContainer("_unique-id_", /* config */);
 
 ## Defining Fluid Containers
```

Comment:
This should move out of fluid-static an into tinylicious-client

Reply:
Added in a link here to the tinylicious-client documentation